eng/Version.Details.xml

```diff
@@ -37,227 +37,231 @@
       <Uri>https://github.com/aspnet/EntityFrameworkCore</Uri>
       <Sha>000000</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.Analyzer.Testing" Version="3.0.0-preview.19059.4">
+    <Dependency Name="Microsoft.AspNetCore.Analyzer.Testing" Version="3.0.0-preview.19059.5">
```

Comment:
19059.5 is the earliest version of Microsoft.JSInterop. Updated the remainder of Extensions package in tandem.

Reply:
It appears you hand-edited, so the corresponding update to `<Sha>` is missing. For future reference, the `darc` command is the way to do this. See https://github.com/aspnet/AspNetCore/pull/6714.